src/manager/BaseMapFishPrintManager.ts

```diff
@@ -204,31 +205,31 @@ export class BaseMapFishPrintManager extends Observable {
    * @type {Object}
    * @protected
    */
-  _layout = {};
+  _layout: any = {};
```

Comment:
This (and others) should be typed.

Reply:
Yes, as stated above, this would require a major rewrite.